manifests/server/config.pp

```diff
@@ -229,8 +229,10 @@
         command     => 'systemctl daemon-reload',
         refreshonly => true,
         path        => '/bin:/usr/bin:/usr/local/bin',
+        require     => File['old-systemd-override'],
```

Comment:
Isn't this redundant? The file resource has a notify on `$systemd_notify` which is this resource. Notify (or on this resource, subscribe) is stronger than require. You can verify it in spec tests.